autodoc.pl

```diff
@@ -1249,10 +1249,7 @@ sub parse_config_h {
                     push @links, $2 if defined $2;
                     push @links, $3 if defined $3;
                     foreach my $link (@links) {
-                        if ($link eq 'INSTALL') {
-                            s/ \b INSTALL \b /C<L<INSTALL>>/xg;
```

Comment:
Why not turn `INSTALL` here into `L<INSTALL>`?

Reply:
INSTALL is not a linkable documentation page (installed module, pod, or script), it is a file that comes with the tarball.

Reply:
Or looking at it the other way, `L<INSTALL>` would be a link to a hypothetical `INSTALL.pm` module on CPAN. (Would you look at that, https://metacpan.org/pod/INSTALL actually exists.)

Reply:
A good demonstration that we have no permissions system for documentation names, and that .pod files contained in releases should be namespaced.